composer.json

```diff
@@ -13,7 +13,7 @@
   },
   "require": {
     "symfony/yaml": "~2.6",
-    "guzzlehttp/guzzle": "~5.0",
```

Comment:
We need `"guzzle/guzzle": "~3.9",` not `"guzzlehttp/guzzle": "~5.0"`. 

I couldn't find any place where `guzzlehttp/guzzle` was used in CPanel so I deleted it.

We do need `"guzzle/guzzle": "~3.9",` because of the reasons you stated which is added in this PR.

I've tested on my dev env. Everything seems to work.